jira.py

```diff
@@ -285,6 +295,20 @@ def create_epic(summary, description, labels=None):      #We build the strcture
 # -> I think for our installation it's 'customfield_10007'
 
 
+# =========================================================================================================================================================
```

Comment:
This part isn't executed anymore... you can ignore everything that comes after `sys.exit(1)`